src/NBitcoin/BIP9Deployments.cs

```diff
@@ -11,6 +11,9 @@ public enum BIP9Deployments : int
 
     public class BIP9DeploymentsParameters
     {
+        /// <summary>Special flag for timeout to indicate always active.</summary>
+        public const long AlwaysActive = -1;
```

Comment:
@mikedennis if you have some spare time to check and apply this to NBitcoin I am sure Nicolas would appreciate it.

Reply:
👍